src/hipscat_import/catalog/file_readers.py

```diff
@@ -141,7 +140,9 @@ def read(self, input_file):
         self.regular_file_exists(input_file)
 
         if self.schema_file:
-            schema_parquet = pd.read_parquet(self.schema_file, dtype_backend="numpy_nullable")
+            schema_parquet = file_io.load_parquet_to_pandas(
+                FilePointer(self.schema_file), dtype_backend="numpy_nullable"
```

Comment:
do the api changes mean that we have to cast any path strings to `FilePointer`s for hipscat functions to work? not that it's bad to be explicit.

Reply:
Functionally, I think the methods might work with a string argument? But because this is typed code, we'll fail type checks.